app/presenters/api_docs/api_reference.rb

```diff
@@ -43,7 +42,9 @@ def field_lengths_summary
   private
 
     def new_path?(path)
-      @highlight_yaml['new_paths'].include?(path) unless @highlight_yaml.empty?
+      return false unless draft_schema_file_exists?
```

Comment:
does highlighting always rely on the draft spec, or should we always show on subsequent minor version bumps 🤔 
I think maybe we just do draft, as minor bumps could include different things past new endpoints